gcc/rust/expand/rust-expand-visitor.h

```diff
@@ -158,7 +158,8 @@ class ExpandVisitor : public AST::ASTVisitor
    * @param values Vector of values to mutate in-place and append into
    */
   void expand_inner_items (std::vector<std::unique_ptr<AST::Item>> &values);
-  void expand_inner_stmts (std::vector<std::unique_ptr<AST::Stmt>> &values);
+  void expand_inner_stmts (AST::BlockExpr &expr,
+			   std::vector<std::unique_ptr<AST::Stmt>> &values);
```

Comment:
```suggestion
  void expand_inner_stmts (AST::BlockExpr &expr);
```

Reply:
Fixed in e145c1eb94d76af6dee7f1ec0bc5f7c92f1d2779